src/FSharp.CodeFormat/HtmlFormatting.fs

```diff
@@ -115,39 +121,48 @@ let rec formatTokenSpans (ctx:FormattingContext) = List.iter (function
       ctx.Writer.Write("</span>")
       
   | Token(kind, body, tip) ->
-      // Generate additional attributes for ToolTip
-      let tipAttributes = 
+    // Generate additional attributes for ToolTip
+    let tipAttributes = 
         match tip with
         | Some(tip) -> ctx.FormatTip tip false formatToolTipSpans
         | _ -> ""
 
-      // Get CSS class name of the token
-      let color = 
+    // Get CSS class name of the token
+    let color = 
         match kind with
-        | TokenKind.Comment -> "c"
-        | TokenKind.Default -> ""
-        | TokenKind.Identifier -> "i"
-        | TokenKind.Inactive -> "inactive"
-        | TokenKind.Keyword -> "k"
-        | TokenKind.Number -> "n"
-        | TokenKind.Operator -> "o"
-        | TokenKind.Preprocessor -> "prep"
-        | TokenKind.String -> "s"
-        | TokenKind.TypeOrModule -> "t"
-        | TokenKind.Function -> "f"
-        | TokenKind.Pattern -> "p"
-        | TokenKind.MutableVar -> "v"
-        | TokenKind.Printf -> "pf"
-        | TokenKind.Escaped -> "e"
-
-      if kind <> TokenKind.Default then
+        | TokenKind.Comment       -> CSS.Comment       
+        | TokenKind.Default       -> CSS.Default       
+        | TokenKind.Identifier    -> CSS.Identifier    
+        | TokenKind.Inactive      -> CSS.Inactive      
+        | TokenKind.Keyword       -> CSS.Keyword       
+        | TokenKind.Number        -> CSS.Number        
+        | TokenKind.Operator      -> CSS.Operator      
+        | TokenKind.Preprocessor  -> CSS.Preprocessor  
+        | TokenKind.String        -> CSS.String        
+        | TokenKind.Module        -> CSS.Module        
+        | TokenKind.ReferenceType -> CSS.ReferenceType 
+        | TokenKind.ValueType     -> CSS.ValueType     
+        | TokenKind.Function      -> CSS.Function      
+        | TokenKind.Pattern       -> CSS.Pattern       
+        | TokenKind.MutableVar    -> CSS.MutableVar    
+        | TokenKind.Printf        -> CSS.Printf        
+        | TokenKind.Escaped       -> CSS.Escaped       
+        | TokenKind.Disposable    -> CSS.Disposable    
+        | TokenKind.TypeArgument  -> CSS.TypeArgument  
+        | TokenKind.Punctuation   -> CSS.Punctuation   
+        | TokenKind.Enumeration   -> CSS.Enumeration   
+        | TokenKind.Interface     -> CSS.Interface     
+        | TokenKind.Property      -> CSS.Property      
+        | TokenKind.UnionCase     -> CSS.UnionCase     
```

Comment:
I'm actually not a huge fan of this code style as it complicates adding new (longer) entries and makes the diff longer in such cases (and "blame" harder)...

But I'll probably accept without reverting this (Just for your info)